src/unstract/sdk/platform.py

```
@@ -130,9 +140,13 @@ def _call_service(
 
             response.raise_for_status()
         except ConnectionError as connect_err:
-            msg = "Unable to connect to platform service. Please contact admin."
-            msg += " \n" + str(connect_err)
-            self.tool.stream_error_and_exit(msg)
+            logger.exception("Connection error to platform service: %s", connect_err)
```

Ruff (0.13.1)

Redundant exception object included in `logging.exception` call

(TRY401)